### PR TITLE
Updating tools/ruvseq tool version(s)

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Hello! This is an automated update of the following tool: **tools/ruvseq**. I created this PR because I think the tool's main dependency is out of date, i.e. there is a newer version available through conda.

Please see https://github.com/planemo-autoupdate/autoupdate for more information.

### CODE REVIEW

File: tools/ruvseq/ruvseq.xml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-        <requirement type="package" version="1.22.1">bioconductor-deseq2</requirement>
-        <requirement type="package" version="1.10.0">bioconductor-tximport</requirement>
-        <requirement type="package" version="1.34.1">bioconductor-genomicfeatures</requirement>
-        <requirement type="package" version="0.8.0">r-ggrepel</requirement>
-        <requirement type="package" version="1.20.2">r-getopt</requirement>
+        <requirement type="package" version="1.32.0">bioconductor-deseq2</requirement>
+        <requirement type="package" version="1.20.0">bioconductor-tximport</requirement>
+        <requirement type="package" version="1.44.0">bioconductor-genomicfeatures</requirement>
+        <requirement type="package" version="0.9.1">r-ggrepel</requirement>
+        <requirement type="package" version="1.20.3">r-getopt</requirement>
```
Comment (mvdbeek) [inline tools/ruvseq/ruvseq.xml:13]:
```suggestion
        <requirement type="package" version="1.20.3">r-getopt</requirement>
        <requirement type="package" version="1.4.4">r-reshape2</requirement>
```

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
There were no changes in ruvseq ... if I clone https://git.bioconductor.org/packages/RUVSeq I only see bioconductor maintenance commits that bump the version number, and the last new commit for ruvseq that actually changes code is from July 2017. Would be nice if bioconductor would only bump versions if there are any changes.

Are there any upsides in merging this @simonbray @bgruening?

**Comment by simonbray** [comment]
I don't know anything about the tool (maybe the updates of the minor requirements are useful?) but if there is really no difference in functionality, I would say not.

**Comment by bgruening** [comment]
> There were no changes in ruvseq ... if I clone https://git.bioconductor.org/packages/RUVSeq I only see bioconductor maintenance commits that bump the version number, and the last new commit for ruvseq that actually changes code is from July 2017. Would be nice if bioconductor would only bump versions if there are any changes.
> 
> Are there any upsides in merging this @simonbray @bgruening?

Mh, that sounds like some work on the Bioconda sides, maybe Bioconductor could annotate that?

**Pull request merged**